build-hidapi.sh

```diff
@@ -26,316 +37,381 @@
       else
-        echo -e "\033[32mOK\033[0m"
+        echo -e "${green}OK${plain}"
         cp linux/.libs/libhidapi-hidraw.so ../../Java/Personal/hid4java/src/main/resources/linux-armel/libhidapi.so
         cp libusb/.libs/libhidapi-libusb.so ../../Java/Personal/hid4java/src/main/resources/linux-armel/libhidapi-libusb.so
     fi
   else
-    echo -e "\033[33mSkipping linux-armel\033[0m"
+    echo -e "${yellow}Skipping linux-armel${plain}"
 fi
-echo -e "\033[32m------------------------------------------------------------------------\033[0m"
+echo -e "${green}------------------------------------------------------------------------${plain}"
 
 # 32-bit ARMv7 hard float (linux-arm)
 if [[ "$1" == "all" ]] || [[ "$1" == "linux" ]] || [[ "$1" == "linux-arm" ]]
   then
-    echo -e "\033[32mBuilding ARMv7 hard float\033[0m"
+    echo -e "${green}Building ARMv7 hard float${plain}"
     dockcross-linux-armv7 bash -c 'sudo dpkg --add-architecture armhf && sudo rm -Rf /var/lib/apt/lists && sudo apt-get update && sudo apt-get --yes install libudev-dev:armhf libusb-1.0-0-dev:armhf gcc-arm-linux-gnueabihf && sudo make clean && sudo ./bootstrap && sudo ./configure --host=arm-linux-gnueabihf CC=arm-linux-gnueabihf-gcc && sudo make'
     if [[ "$?" -ne 0 ]]
       then
-        echo -e "\033[31mFailed\033[0m - Removing damaged targets"
+        echo -e "${red}Failed${plain} - Removing damaged targets"
         rm ../../Java/Personal/hid4java/src/main/resources/linux-arm/libhidapi.so
       else
-        echo -e "\033[32mOK\033[0m"
+        echo -e "${green}OK${plain}"
         cp linux/.libs/libhidapi-hidraw.so ../../Java/Personal/hid4java/src/main/resources/linux-arm/libhidapi.so
         cp libusb/.libs/libhidapi-libusb.so ../../Java/Personal/hid4java/src/main/resources/linux-arm/libhidapi-libusb.so
     fi
   else
-    echo -e "\033[33mSkipping linux-arm\033[0m"
+    echo -e "${yellow}Skipping linux-arm${plain}"
 fi
-echo -e "\033[32m------------------------------------------------------------------------\033[0m"
+echo -e "${green}------------------------------------------------------------------------${plain}"
 
 # OS X environments
 
-# Darwin
+# Darwin x86_64
 if [[ "$1" == "all" ]] || [[ "$1" == "osx" ]] || [[ "$1" == "darwin" ]]
   then
-    echo -e "\033[32mBuilding OS X Darwin\033[0m"
+    echo -e "${green}Building OS X Darwin${plain}"
     make clean
     ./bootstrap
     ./configure
-    make
+    make CFLAGS="-arch x86_64"
     if [[ "$?" -ne 0 ]]
       then
-        echo -e "\033[31mFailed\033[0m - Removing damaged targets"
+        echo -e "${red}Failed${plain} - Removing damaged targets"
         rm ../../Java/Personal/hid4java/src/main/resources/darwin/libhidapi.dylib
       else
-        echo -e "\033[32mOK\033[0m"
+        echo -e "${green}OK${plain}"
         cp mac/.libs/libhidapi.0.dylib ../../Java/Personal/hid4java/src/main/resources/darwin/libhidapi.dylib
```

Comment:
Does this need to change to `darwin_x86-64`?

Reply:
Answering my own question, yes, [it should change](https://github.com/java-native-access/jna/commit/ab2cb98cb5b267d964d465ad95d8161808f18b97), but JNA was courteous to fallback to the old location if missing.  Will patch and resolve this conversation.